homeassistant/components/bond/light.py

```diff
@@ -72,6 +82,17 @@ async def async_setup_entry(
         if DeviceType.is_light(device.type)
     ]
 
+    platform.async_register_entity_service(
+        SERVICE_SET_LIGHT_STATE_BELIEF,
+        {
+            vol.Optional(ATTR_BRIGHTNESS): vol.All(
+                vol.Number(scale=0), vol.Range(0, 100)
+            ),
+            vol.Optional(ATTR_POWER_STATE): vol.All(cv.boolean),
+        },
+        "async_set_state_belief",
+    )
```

Comment:
It looks like the arguments are mutually exclusive.

In that case, split this into two separate services so you can make the arguments required.

Reply:
Is there a way to make this require at least one of them with vol?